Sources/MultiSlider+Internal.swift

```diff
@@ -271,7 +271,8 @@ extension MultiSlider {
     private func position(marker: UIView, at value: CGFloat) {
         guard let containerView = marker.superview else { return }
         containerView.removeFirstConstraint { $0.firstItem === marker && $0.firstAttribute == .center(in: orientation) }
-        let relativeDistanceToMax = (maximumValue - value) / (maximumValue - minimumValue)
+        let minMaxValueDifference = maximumValue - minimumValue
+        let relativeDistanceToMax = (maximumValue - value) / (minMaxValueDifference.isZero ? 1 : minMaxValueDifference)
```

Comment:
Thanks! When the slider size is 0, the distance should be 0 and not 1:

```suggestion
        let relativeDistanceToMax = (maximumValue - value) / (minMaxValueDifference.isZero ? 0 : minMaxValueDifference)
```

Reply:
Zero will crash, because swift cannot divide to zero

Reply:
There is no division by `relativeDistanceToMax` - please run it and see if it works for you.

Reply:
`(minMaxValueDifference.isZero ? 0 : minMaxValueDifference)`  can be zero 
`(maximumValue - value) / (minMaxValueDifference.isZero ? 0 : minMaxValueDifference)`

Reply:
Right! Sorry, my mistake.

Reply:
So it will look like this
```suggestion
        let relativeDistanceToMax = minMaxValueDifference.isZero ? 1 : (maximumValue - value) / minMaxValueDifference
```

Reply:
Yes.